### ISSUE TITLE
Adottare un data-picker per i form

### ISSUE THREAD

**Issue by idlweb** [idlweb/openordini#25]

**Comment by idlweb** [comment]
birth_date = forms.DateField(required=True, label=_("Birth date"), widget=forms.widgets.DateInput(format="%d/%m/%Y", attrs=
                                {
                                    'class':'datepicker'
                                }), help_text=u"Usa il formato gg/mm/aaaa")
+

<script>
  $(function() {
    $( ".datepicker" ).datepicker({
      changeMonth: true,
      changeYear: true,
      yearRange: "1900:2012",
      // You can put more options here.

    });
  });
  </script>

**Comment by idlweb** [comment]
per la traduzione:
 <script src="//ajax.googleapis.com/ajax/libs/jqueryui/1.11.1/i18n/jquery-ui-i18n.min.js"></script>
e

    <script type="text/javascript">
      $(document).ready(function(){
         
         $.datepicker.setDefaults(
            $.extend(
              {'dateFormat':'dd-mm-yy'},
              $.datepicker.regional['it']
            )
          );

         $( ".datepicker" ).datepicker({
            changeMonth: true,
            changeYear: true,
            yearRange: "1900:2012",
            // You can put more options here.
          });
      });
  </script>
